bot.py

```diff
@@ -95,13 +95,13 @@
     if message.content.lower() == "odiekick":
         response = "http://66.media.tumblr.com/tumblr_lq5hk354UG1qbqwr6o1_400.gif"
         await message.channel.send(response)
-        
-    if message.content.lower() == "garflinks":
+    
+    if "garflinks" in message.content.lower():
```

Comment:
please keep this line the same as all of the other command if statements before it. the lasagne one is an exception